src/storage/adapter/wasm.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use gloo_storage::LocalStorage;
+
+use super::StorageAdapter;
+
+/// The storage id.
+pub const STORAGE_ID: &str = "Wasm";
+
+/// Wasm storage adapter using the browser local storage
+#[derive(Debug)]
+pub struct WasmAdapter(LocalStorage);
+
+impl WasmAdapter {
+    /// Initialises the storage adapter.
+    pub fn new() -> crate::Result<Self> {
+        Ok(Self(LocalStorage::new()))
+    }
+}
+
+#[async_trait::async_trait]
+impl StorageAdapter for WasmAdapter {
+    async fn get(&self, key: &str) -> crate::Result<String> {
```

Comment:
Missing doc, also can you please copy paste docs from the other adapters, I slightly changed them.